.github/workflows/ci.yml

```diff
@@ -209,6 +209,11 @@ jobs:
             xvfb: true
             node: 18
             suite: firefox-bidi
+          - name: firefox-headless
+            machine: macos-latest
+            xvfb: false
+            node: 18
+            suite: firefox-headless
```

Comment:
I wonder if also the `headful` variant should be run on Windows? Looks like it's also not done for Chrome.